src/Microsoft.OpenApi/Validations/OpenApiValidator.cs

```diff
@@ -39,19 +50,19 @@ public OpenApiValidator(ValidationRuleSet ruleSet = null)
         /// Execute validation rules against an <see cref="OpenApiInfo"/>
         /// </summary>
         /// <param name="item">The object to be validated</param>
-        public override void Visit(OpenApiInfo item) => Validate(item);
+        public override void Visit(OpenApiInfo item) => Validate("info",item);
```

Comment:
There are constants for the property names ("info", "contact", etc.) in OpenApiConstants class somewhere.

Reply:
Oops. Sorry. I got confused by the GitHub PR UI. This is outdated, so please ignore.